charts/__tests__/gardener-dashboard/application/rolebinding.spec.js

```diff
@@ -0,0 +1,50 @@
+//
+// SPDX-FileCopyrightText: 2023 SAP SE or an SAP affiliate company and Gardener contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+'use strict'
+
+const { basename } = require('path')
+const { helm } = fixtures
+
+const renderTemplates = helm.renderTemplatesFn('gardener-dashboard', 'charts', basename(__dirname))
+
+describe('gardener-dashboard', function () {
+  describe('rolebinding', function () {
```

Comment:
```suggestion
describe('gardener-dashboard', () => {
  describe('rolebinding', () => {
```

Reply:
When we moved from [Mocha](https://mochajs.org/) to [Jest](https://jestjs.io/) we have not change every function to an arrow function but for all new tests we should use arrow functions in Jest.